client-rpc/src/rpc/staking_rpc.rs

```diff
@@ -44,6 +44,9 @@ pub trait StakingRpc: Send + Sync {
         to_address: String,
         view_keys: Vec<String>,
     ) -> Result<String>;
+
+    #[rpc(name = "staking_unjail")]
+    fn unjail(&self, request: WalletRequest, unjail_address: String) -> Result<String>;
```

Comment:
can you base it off the latest master? this was already merged: https://github.com/crypto-com/chain/commit/ce3c976cf1d7004eae0ab4ed9c1b1d2875708168 so this PR will only show things that changed or are new

Reply:
ok